crates/ruff_linter/src/rules/airflow/rules/dag_schedule_argument.rs

```diff
@@ -9,16 +9,17 @@ use crate::checkers::ast::Checker;
 
 /// ## What it does
 /// Checks for a `DAG()` class or `@dag()` decorator without an explicit
-/// `schedule` parameter.
+/// `schedule` (or `schedule_interval` for Airflow 1) parameter.
 ///
 /// ## Why is this bad?
-/// The default `schedule` value on Airflow 2 is `timedelta(days=1)`, which is
-/// almost never what a user is looking for. Airflow 3 changes this the default
-/// to *None*, and would break existing DAGs using the implicit default.
+/// The default `schedule` value on Airflow 2 and `schedule_interval` on Airflow 1
+/// is `timedelta(days=1)`, which is almost never what a user is looking for.
+/// Airflow 3 changes this the default to *None*, and would break existing dags
```

Comment:
"Airflow 3 changed the default value to `None`, ..."

Reply:
Updated. Thanks!